opentext-page.html

```diff
@@ -0,0 +1,482 @@
+            <button type="submit">Watch the webinar</button>
+        </div>
+        <div class="slide-show">
+             <!-- div has two columns willl add second column when learn about slideshows -->
+            <img src="#img3" alt="image of astronaut">
+            <h2>Defy gravity</h2>
+            <p>Rethink the complexity of your supply chain operations</p>        
+            <button type="submit">Learn more</button>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Cloud login</span>
+                </div>
+            </div>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Get Support</span>
+                </div>
+            </div>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Developer</span>
+                </div>
+            </div>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Contact us</span>
+                </div>
+            </div>
+        </div>
+        <div>
+            <a href="url">Upgrade to the latest OpenText sofware faster and reduce risk</a>
+        </div>
+        <div class="container-3">
+            <div class="row">
+                <div class="column">
+                    <h3>Rethink the Future of Business</h3>
+                    <p>Adapt to global change and technology disruption by rethinking business strategy and seizing new digital capabilities in the cloud.</p>
+                    <button type="submit">Read the CEO white paper</button>
+                </div>
+            </div>
+        </div>
+        <div class="container-3">
+            <div class="row">
+                <div class="column">
+                    <ul>
+                        <li><a href="#Content">Content Cloud</a></li>
+                        <li><a href="#Experience">Experience Cloud</a></li>
+                        <li><a href="#Developer">Developer Cloud</a></li>
+                    </ul>
+                </div>
+            </div>        
+        </div>
+        <div class="container-3">
+            <div class="row">
+                <div class="column">
+                    <ul>
+                        <li><a href="#Business">Business Network Cloud</a></li>
+                        <li><a href="#Security">Security & Protection Cloud</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>    
+        <div class="client">
```

Comment:
Requires also container class here, and applies to all other cases below with a div w/ client class.